### PR TITLE
Expose greenlet number as environment variable

### PR DESCRIPTION
**Pull request by leo-p**
Expose a new environment variable `DEEPOMATIC_CLI_ADD_IMAGES_CONCURRENCY` which defaults to 5.

### CODE REVIEW

File: deepomatic/cli/cmds/feedback.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 ###############################################################################
 
-GREENLET_NUMBER = 10
+GREENLET_NUMBER = int(os.getenv('DEEPOMATIC_CLI_ADD_IMAGES_GREENLET_NUMBER', 10))
```
Comment (thomas-riccardi) [inline deepomatic/cli/cmds/feedback.py]: maybe instead name with `parallel` somewhere: greenlet number is really an internal implementation detail
Suggestion: `DEEPOMATIC_CLI_ADD_IMAGES_PARALLEL_LEVEL`

Comment (thomas-riccardi) [inline deepomatic/cli/cmds/feedback.py]: maybe also change the default value to 1 for now?

Comment (maingoh) [reply deepomatic/cli/cmds/feedback.py]: It's okay, it is a quite internal env var. I prefer `DEEPOMATIC_CLI_ADD_IMAGES_CONCURRENCY`.

Comment (maingoh) [reply deepomatic/cli/cmds/feedback.py]: `1` will be really slow, I think 5 is a good default.

**Pull request merged**